xbmc/video/dialogs/GUIDialogCMSSettings.cpp

```diff
@@ -0,0 +1,243 @@
+/*
+ *      Copyright (C) 2005-2014 Team XBMC
```

Comment:
Thank you! I just set it to Team Kodi since most of the file is copied directly from another dialog.